README.md

```diff
@@ -82,6 +82,14 @@ For example code using Winit with WebAssembly, check out the [web example]. For
 information on using Rust on WebAssembly, check out the [Rust and WebAssembly
 book].
 
+To run the web example:
+
+1. Ensure you have bash and python 3 installed.
```

Comment:
Requiring `bash` for everyone, even on Windows, feels a bit weird, but I suppose everyone using git has access to "git bash". As for listing python 3 as a hard dependency, well, that seems like a slight lie, as installing `basic-http-server` would seem to circumvent the need for python.

Reply:
tbh I have no idea how well it runs on windows.
I'm just taking from the wgpu docs which seems to suggest it works fine as long as you have bash https://github.com/gfx-rs/wgpu/wiki/Running-on-the-Web-with-WebGPU-and-WebGL